docs/administration/back_office/search_suggestion.md

```diff
@@ -0,0 +1,157 @@
+---
+description: Customize search suggestion configuration and sources.
+---
+
+# Customize search suggestion
+
+In the Back Office, when you start typing in the search field on the top bar, suggestions about what you could be looking for show up directly under the field.
+For more information about using this feature to search for content, see [user documentation]([[= user_doc =]]/search/search_for_content).
+
+## Configuration
+
+By default, suggestions start showing up after the user types in at least 3 characters, and 5 suggestions are presented.
+This can be changed with [SiteAccess aware dynamic configuration](dynamic_configuration.md), by setting the following [scoped](multisite_configuration.md#scope) parameters:
+
+```yaml
+parameters:
+    ibexa.site_access.config.<scope>.search.suggestion.min_query_length: 3
+    ibexa.site_access.config.<scope>.search.suggestion.result_limit: 5
+```
+
+## Add custom suggestion source
+
+You can add a suggestion source by listening or subscribing to `Ibexa\Contracts\Search\Event\BuildSuggestionCollectionEvent`.
+During this event, you can add, remove, or replace suggestions by updating its `SuggestionCollection`.
+After this event, the suggestion collection is sorted by score and truncated to a number of items set in [`result_limit`](#configuration).
+
+!!! tip
+
+    You can list listeners and subscribers with the following command:
+    ``` shell
+    php bin/console debug:event BuildSuggestionCollectionEvent
+    ```
+
+The following example is boosting Product suggestions.
+It's a subscriber that passes after the default one (because priority is set zero), adds matching products at a score above the earlier Content suggestions, and avoids duplicates.
```

Comment:
```suggestion
It's a subscriber that passes after the default one (because priority is set to zero), adds matching products at a score above the earlier Content suggestions, and avoids duplicates.
```

Reply:
done in 83c8511101b3dd044b09b7728041054825c76528